brave-http/src/main/java/com/github/kristofa/brave/http/HttpClientRequestAdapter.java

```diff
@@ -8,50 +8,54 @@
 import com.twitter.zipkin.gen.Endpoint;
 import zipkin.TraceKeys;
 
-import java.net.URI;
-import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
 
 public class HttpClientRequestAdapter implements ClientRequestAdapter {
 
-    private final HttpClientRequest request;
+    private final HttpClientRequest clientRequest;
```

Comment:
since this code is doing things for acts like saving allocations, might be worth profiling the difference between just making this protected vs making an overridable method.

If all you need is to access the field, I'd probably make it protected, keeps the scope down, as I don't think we should be changing the type of `HttpClientRequest` returned consumed by this adapter without explaining why.